docs/blog/20230903-confused-Techie-pulsars-ci.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Pulsar's CI"
+author: confused-Techie
+date: 2023-09-02
+category:
+  - dev
+tag:
+  - ci
+---
+
+How Pulsar's CI has recently changed.
+
+<!-- more -->
+
+CI, or Continuous Integration, is a rather broad term used to describe the DevOps process of continuously (oftentimes on every change) building and testing a piece of software. In Pulsar's case this means two big things:
+
+- Testing Pulsar and all of it's core packages
+- Building Pulsar binaries, and even then running tests using those built binaries.
+
+To better understand how drastically things have changed, lets quickly take a look how Pulsar's CI has looked up until this month since nearly the beginning of the project.
```

Comment:
```suggestion
To better understand how drastically things have changed, let's quickly take a look how Pulsar's CI has looked up until this month since nearly the beginning of the project.
```